source/stylesheets/scss/furatto/_badges.scss

```diff
@@ -0,0 +1,80 @@
+//Badges
+
+//
+//@variables
+//
+
+@import "functions";
+
+$badge-text-font-size: px-to-rems(16) !default;
+$badge-text-color: #B6B6B6 !default;
+$badge-background-color: #FF4081 !default;
+$badge-width: px-to-rems(22) !default;
+$badge-height: $badge-width !default;
+$badge-font-weight: 600 !default;
+$badge-font-size: 75% !default;
+$badge-color: #FFF !default;
+$badge-right-position: px-to-rems(-22) !default;
+$badge-top-position: px-to-rems(-11) !default;
+$badge-no-background-color: #212121 !default;
+
+
+//
+// @mixin
+//
+// We use this mixin to add badges variations
+//
+// .badge {
+//   &.primary {
+//    @include badge-background-variation($background-color: #3F51B5);
+//   }
+// }
+//
+@mixin badge-background-variation($background-color,
+                                  $color: #FFF,
+                                  $font-weight: 600) {
+  &:after {
+    background: $background-color;
+    color: $color;
+    font-weight: $font-weight;
+  }
+}
+
+.badge {
+  color: $badge-text-color;
+  display: inline-block;
+  font-size: $badge-text-font-size;
+  margin-right: 24px;
+  position: relative;
+  white-space: nowrap;
+
+  &:after {
+    @include align-content(center);
+    @include align-items(center);
+    @include border-radius(50%);
+    @include display-flex;
+    @include flex-wrap(wrap);
+    @include flex-direction(row);
+    @include justify-content(center);
+
+    -webkit-box-orient: horizontal;
```

Comment:
Properties should be ordered background, -webkit-box-direction, -webkit-box-orient, color, content, font-size, font-weight, height, position, right, top, width